### DISCUSSION TITLE
sh start-all.sh的过程中一直报错，请告诉下是什么原因

### DISCUSSION THREAD

**Discussion by zhouxi-24hours** [WeBankFinTech/DataSphereStudio#994] [Q&A]
报错信息：
We will start all linkis applications, it will take some time, please wait
<-------------------------------->
Begin to start mg-eureka
Is remote execution:sh /home/hadoop/linkis/sbin/linkis-daemon.sh restart mg-eureka
Bad port '192.168.5.48172.17.0.1'
Failed to  + End to start mg-eureka
Failed to  + start Linkis

网卡确定是设置成了单网卡
[hadoop@tele-hearing bin]$ hostname -I
192.168.5.48

**Comment by dachn** [comment]
if you cmd"ifconfig",you'll get more than one network interface card ，one of those is 172.17.0.1,(maybe your computer have installed docker），you should shutdown them，and follow this：
https://github.com/WeBankFinTech/DataSphereStudio-Doc/blob/main/zh_CN/%E5%AE%89%E8%A3%85%E9%83%A8%E7%BD%B2/DSS%26Linkis%E4%B8%80%E9%94%AE%E9%83%A8%E7%BD%B2%E6%96%87%E6%A1%A3%E5%8D%95%E6%9C%BA%E7%89%88.md
